---
title: Article summary VI
---

## Summary
Using a variety of techniques, [Parvanov et al.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2821451/), [Baudat et al.](https://science.sciencemag.org/content/327/5967/836.full), and [Myers et al.](https://science.sciencemag.org/content/327/5967/876.full) described the major discovery of PR domain zinc finger protein 9 (PRDM9) as a key factor controlling the distribution of preferred chromosomal locations, known as “hotspots”, for recombination. The groups of [Bernard de Massey](https://science.sciencemag.org/content/327/5967/836.full) and [Kenneth Paigen](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2821451/)), working in mice, independently identified a region on chromosome 17 that functioned *in trans* to activate hotspots in distant locations. This transactivation locus was reduced to a tiny 181-kb region containing only four genes, of which one, PRDM9, became an attractive candidate for genes involved in mammalian hot spot regulation through chromatin modification: its destruction during early meiosis prevents prophase processes leading to sterility, and importantly, PRDM9 contains a conserved central domain with H3K4 methyltransferase activity. Baudat et al. also identify human allelic variants within Prdm9 that differed in the frequency at which they used hotspots. Meanwhile, the third group of researchers took a completely different approach to investigate what specifies human recombination hotspots. By systematically searching for sequence motifs present in the hotspots associated with phase 2 HapMap, [Myers and colleagues](https://science.sciencemag.org/content/327/5967/876.full) identified a 13 bp degenerate motif (CCNCCNTNNCCNC) that is critical for determining 40% of the recombination activity in all known meiotic hotspots, the rapidly evolving zinc-finger protein PRDM9 binds to this motif and that sequence changes in the protein may be responsible for hotspot differences between species.

## Reaction
The dramatic feature exposed by these studies is of the relative fluidity with which recombination distributions can be altered by combining in one protein (PRDM9) and epigenetic marking activity (H3K4me3) with a rapidly diverging DNA binding domain. The differential binding of this protein to different human alleles suggests that this protein interacts with specific DNA sequences. PDRM9 functions in the determination of recombination loci within the genome and may be a significant factor in the genomic differences between closely related species. The finding of the three studies should help us to understand the impact of recombination on aneuploidy in oocytes. Moreover, it opens the door to understand the balance of successful gamete formation and maintenance of genetic diversity. Yet many interesting questions are unresolved. The interacting partners of PRDM9 and how do they work together to direct recombination is not clear. How does PRDM9 decide which sites to use? PRDM9 explains only an estimated 18% of individual variation in hotspot usage, so what do other genes contribute to this variation? What are the interactions between human genes which were identified as genetic determinants of the number of recombinations per meiosis and PRDM9? Is there a minimum number of recombinations required for proper meiosis? The answers to these questions will significantly advance our understanding of the clustering of recombination in the genome and have more clinically meaningful impacts. 

## Questions
1. What is the significance of the DNA binding specificity of PRDM9, and why is it evolving so rapidly?
2. Does PRDM9 specify all or just some hotspots?
3. What are the clinical impacts and long-term effects of these studies? 
  











